ingestion/src/metadata/ingestion/source/database/postgres/__init__.py

```diff
@@ -0,0 +1,19 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+Module constants
+"""
+
+
+POSTGRES_GET_DATABASE = """
```

Comment:
same as mssql. Also for postgres I believe we already have a queries module

Reply:
sure.